system/TestAuditSuite.py

```diff
@@ -0,0 +1,184 @@
+import pytest
+from system.utils import *
+
+
+@pytest.mark.usefixtures('docker_setup_and_teardown')
+class TestAuditSuite:
+
+    @pytest.mark.asyncio
+    async def test_case_restart_one_node(self, pool_handler, wallet_handler, get_default_trustee):
+        trustee_did, _ = get_default_trustee
+        hosts = [testinfra.get_host('ssh://node{}'.format(i)) for i in range(1, 8)]
+        for i in range(15):
+            await nym_helper(pool_handler, wallet_handler, trustee_did, random_did_and_json()[0], None, None, None)
+        output = hosts[5].check_output('systemctl restart indy-node')
+        print(output)
+        for i in range(30):
+            await nym_helper(pool_handler, wallet_handler, trustee_did, random_did_and_json()[0], None, None, None)
+        await eventually_positive(check_ledger_sync, is_self_asserted=True)
+        primary1, alias, target_did = await get_primary(pool_handler, wallet_handler, trustee_did)
+        output = testinfra.get_host('ssh://node{}'.format(primary1)).check_output('systemctl stop indy-node')
+        print(output)
+        primary2 = await wait_until_vc_is_done(primary1, pool_handler, wallet_handler, trustee_did)
+        assert primary2 != primary1
+        for i in range(15):
+            await nym_helper(pool_handler, wallet_handler, trustee_did, random_did_and_json()[0], None, None, None)
+        output = hosts[5].check_output('systemctl restart indy-node')
+        print(output)
+        for i in range(30):
+            await nym_helper(pool_handler, wallet_handler, trustee_did, random_did_and_json()[0], None, None, None)
+        output = testinfra.get_host('ssh://node{}'.format(primary1)).check_output('systemctl start indy-node')
+        print(output)
```

Comment:
This looks like a perfect candidate for moving into some helper function, like
```python
def get_node(node_id: int):
    return testinfra.get_host('ssh://node{}'.format(node_id))

def start_node_service(node_id: int):
    output = get_node(node_id).check_output('systemctl start indy-node')
    print(output)
```

Or even make some helper class like
```python
class TestNode:
    def __init__(self, node_id: int):
        self._host = testinfra.get_host('ssh://node{}'.format(node_id))

    def run(self, command: str):
        output = self._host.check_output(command)
        print(output)

    def start_service(self):
        self.run('systemctl start indy-node')
```

Reply:
Done.